tests/functional/RemoteWebDriverFindElementTest.php

```diff
@@ -61,4 +61,22 @@ public function testShouldFindMultipleElements()
         $this->assertCount(5, $elements);
         $this->assertContainsOnlyInstancesOf(RemoteWebElement::class, $elements);
     }
+
+    public function testEscapeCssSelector()
+    {
+        if (!getenv('GECKODRIVER') === '1') {
+            $this->markTestSkipped('Invalid class identifiers are not supported by the legacy protocol');
```

Comment:
I would not call it `invalid`. The `class` is not invalid (it is not a valid identifier indeed, and so requires escaping, but the message is not clear IMO).

and if this only applies to classes, maybe the test should be split (to skip only the one about classes)

Reply:
I tried to escape identifiers for the legacy protocol too, but it doesn't work for other reasons. It applies to all identifiers.

wdyt of "CSS selectors containing special characters are not supported by the legacy protocol"?